core/tauri-macros/src/lib.rs

```diff
@@ -151,3 +154,35 @@ pub fn do_menu_item(input: TokenStream) -> TokenStream {
   let tokens = parse_macro_input!(input as menu::DoMenuItemInput);
   menu::do_menu_item(tokens).into()
 }
+
+/// Convert a .png or .ico icon to an Image
+/// for things like `tauri::tray::TrayIconBuilder` to consume,
+/// relative paths are resolved from `CARGO_MANIFEST_DIR`, not current file
+///
+/// ### Examples
+///
+/// ```ignore
+/// TrayIconBuilder::new().icon(icon_image!("./icons/32x32.png")).build().unwrap();
+/// ```
+///
+/// Note: this stores the image in raw pixels to the final binary,
+/// so keep the icon size (width and height) small
+/// or else it's going to bloat your final executable
+#[proc_macro]
+pub fn icon_image(tokens: TokenStream) -> TokenStream {
```

Comment:
this should just be called `image` since it will create an `Image` struct or maybe call it `include_image` or `static_image`

Reply:
I'm using `icon_image` here is because this things should only be used on small images for icon purposes, name can change, just some thoughts on why it's so called

Reply:
I think `include_image` is probably the best name here and the note in the documentation should be enough for size concerns